javascript.js

```diff
@@ -0,0 +1,134 @@
+
+/* elements & variables */
+var navBarProfilePic = document.getElementsByClassName("navBar__bgPicture")[0];
+var navBarDownloadCV = document.getElementsByClassName("navBar__CVbtn")[0];
+var professionalSection = document.getElementById("professional");
+var homeDownloadCV = document.getElementsByClassName("home__downloadCV")[0];
+var SideBar = document.getElementById("navBar");
+var porcentageBars = document.getElementsByClassName("professional__percentBar");
+var verticalLine = document.getElementsByClassName("experince__line")[0];
+var vectorsPoints = document.getElementsByClassName("experince__point");
+var flipButton = document.getElementsByClassName("flip__btn");
+var awardsSection = document.getElementById("awardsInterest");
+var navBarScrolled = false;
+
+/* FUNCTIIONS */
+
+/* General Functions */
+function detectmob() { 
+    if( navigator.userAgent.match(/Android/i)
+    || navigator.userAgent.match(/webOS/i)
+    || navigator.userAgent.match(/iPhone/i)
+    || navigator.userAgent.match(/iPad/i)
+    || navigator.userAgent.match(/iPod/i)
+    || navigator.userAgent.match(/BlackBerry/i)
+    || navigator.userAgent.match(/Windows Phone/i)
+    ){
+        return true;
+    } 
+    else {
+        return false;
+    }
+}
+/* Navigation Functions */
+function navBarSlide() {
+    if (document.documentElement.scrollTop >= (homeDownloadCV.offsetTop + homeDownloadCV.clientHeight) && !navBarScrolled) {
+        navBarScrolled = true;
+        removeNavClasses();
+    }
+    if (document.documentElement.scrollTop < (homeDownloadCV.offsetTop + homeDownloadCV.clientHeight) && navBarScrolled) {
+        navBarScrolled = false;
+        addNavClasses(); 
+    }
+}
+
+function removeNavClasses () {
+    navBarProfilePic.classList.remove("navBar__profilePic--offview");
+    navBarDownloadCV.classList.remove("navBar__CVbtn--offview");
+}
+
+function addNavClasses () {
+    navBarProfilePic.classList.add("navBar__profilePic--offview");
+    navBarDownloadCV.classList.add("navBar__CVbtn--offview");
+}
+
+function openNav() {
+    removeNavClasses();
+    SideBar.classList.add("navBar__sideBar--open");
+}
+
+function closeNav() {
+    addNavClasses();
+    SideBar.classList.remove("navBar__sideBar--open");
+}
+
+function slideEventListener() {
+    if (!detectmob() || window.innerWidth > 768) {
+        document.addEventListener("scroll", navBarSlide);
+    } 
+    else {
+        document.removeEventListener("scroll", navBarSlide);
+    }
+}
+
+function updateSideBar () {
```

Comment:
You can try this, if it works is better is just one line 

```js
window.innerWidth <= 768 ? closeNav() : navBarSlide();
```

Reply:
Do you mean instead of using a function, should I use it like this?
`window.addEventListener("resize", window.innerWidth <= 768 ? closeNav() : navBarSlide());
`